### ISSUE TITLE
build failed (Could not resolve project :firebase_core_web.) after upgrading 1.2.13+hotfix 5

### ISSUE THREAD

**Issue by Xgamefactory** [FirebaseExtended/flutterfire#1661]
```


FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':firebase_core:compileDebugJavaWithJavac'.
> Could not resolve all task dependencies for configuration ':firebase_core:debugCompileClasspath'.
   > Could not resolve project :firebase_core_web.
     Required by:
         project :firebase_core
      > Unable to find a matching configuration of project :firebase_core_web:
          - None of the consumable configurations have attributes.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.


why firebase_core_web is required anyway it is mobile app project.
* Get more help at https://help.gradle.org

BUILD FAILED in 1s
Gradle task assembleDebug failed with exit code 1
Exited (sigterm)
```

we are using lates available firebase_core plugin firebase_core: **^0.4.3+1**

```
[flutter] flutter doctor -v
[✓] Flutter (Channel stable, v1.12.13+hotfix.5, on Mac OS X 10.14.6 18G103, locale en-TR)
    • Flutter version 1.12.13+hotfix.5 at /Users/fersmart/SDK_all/flutter
    • Framework revision 27321ebbad (7 days ago), 2019-12-10 18:15:01 -0800
    • Engine revision 2994f7e1e6
    • Dart version 2.7.0

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/fersmart/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3, Build version 11C29
    • CocoaPods version 1.7.5

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 42.1.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] VS Code (version 1.41.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.7.1

[✓] Connected device (3 available)
    • SM G965F        • 24194f98b60c7ece          • android-arm64 • Android 9 (API 28)
    • SM J415FN       • 4c22d384                  • android-arm   • Android 9 (API 28)
    • iPhone Fersmart • 00008020-000E49DA0CD8003A • ios           • iOS 13.2.2

• No issues found!
exit code 0
```

**Comment by 2ZeroSix** [comment]
until this issue is resolved you can add this into your pubspec.yaml dependencies to fix build errors:
```firebase_core_web: ^0.1.1+1```

**Comment by 2ZeroSix** [comment]
seems to be fixed in `firebase_core: ^0.4.3+2`
